{{cookiecutter.project_slug}}/.github/ISSUE_TEMPLATE.md

```diff
@@ -1,27 +1,18 @@
+## General
+
 * {{ cookiecutter.project_name }} version:
 * Python version:
 * Operating System:
+* Did you install pyfar via pip:
 
-### Description
+## Description
 
 Describe what you were trying to get done.
 Tell us what happened, what went wrong, and what you expected to happen.
 
-### What I Did
+## What I Did
 
 ```
 Paste the command(s) you ran and the output.
 If there was a crash, please include the traceback here.
 ```
-
-## Labels
```

Comment:
Why are we removing this?

Reply:
i just copied from [pyfar](https://github.com/pyfar/pyfar/tree/main/.github)